src/mlpack/methods/reinforcement_learning/q_networks/dueling_dqn.hpp

```diff
@@ -0,0 +1,124 @@
+/**
+ * @file methods/reinforcement_learning/q_networks/dueling_dqn.hpp
```

Comment:
Great, I guess this is WIP, and you will add a test as well?

Reply:
Yeah, I add as soon as the layout is ready :smile: